### ISSUE TITLE
Reviewing the "Resources List" page

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#738]
## Overview
Following up from #645 

I just need a few more tweaks to make these cards final.

<img width="320" alt="Screenshot 2021-06-09 at 14 05 22" src="https://user-images.githubusercontent.com/2813677/121359821-d447e880-c96e-11eb-9dbd-38dc5127e51c.png">

## Conditions of Satisfaction
- [ ] Remove the resource type tagging
- [ ] Add the possibility of having a thumbnail to the card
- [ ] Restyle the "available offline"

## Resources
- Reference design: [InVision](https://projects.invisionapp.com/share/52Y02L9YTRF#/screens/429853508_Resources_List)

**Comment by pedro16v** [comment]
Hi @andrewwapling @PeteCoward @Juliana-t . We'll try to slide this in on the next opportunity but in the meantime, a few questions.

1. For @Juliana-t — @andrewwapling [has mentioned](https://github.com/catalpainternational/canoe/issues/645#issuecomment-856428130) that the resource type tags have been deprecated. Can you confirm?
2. For @PeteCoward
  2.1. Will these thumbnails be added by picking a file on the "Card image" field while adding a new Resource? 
  2.2. Are video thumbnails here an option?

**Comment by Juliana-t** [comment]
@pedro16v 
1. Yes - unless there is any hard requirement for any program / legacy issues for having this here
2. 
2.1 On wagtail I can select an image for a resource entry (I have been doing that for Kumul Helt Skul) though it is not being shown on the app. Maybe it is just a matter of wiring that upload from Wagtail to the app/FE

**Comment by andrewwapling** [comment]
@pedro16v @PeteCoward let me know if you require any additional design support on this issue.

**Comment by pedro16v** [comment]
Looking good @megganeturner !

I only noticed one glitch while adding a thumbnail to a resource. Note how the image overflows from the container. Could we confine it so that it can't use more than 100% of the available row height?

Image overflows | I duplicated the element so it's easier to tell
--- | ---
<img width="375" alt="Screenshot 2021-07-13 at 09 24 49" src="https://user-images.githubusercontent.com/2813677/125426372-4168b9cc-829c-4c2c-bdcc-fea12265d188.png"> | <img width="375" alt="Screenshot 2021-07-13 at 09 24 58" src="https://user-images.githubusercontent.com/2813677/125426361-0e617608-8593-495c-8d81-7cd559dcefe3.png">
